### ISSUE TITLE
@walletconnect NPM version 1.0.17 missing build-registry script

### ISSUE THREAD

**Issue by RyRy79261** [WalletConnect/walletconnect-monorepo#317]
## Description 

The current published version on NPM is version 1.0.17, which doesn't match the github releases.

This version includes, as part of the `qrcode-modal` package's build script, a call to `./scripts/build-registry.js` which is not present in the repo. 

This leads to crashes in any package making use of this version on install. 

```
error .../node_modules/@walletconnect/qrcode-modal: Command failed.
Exit code: 1
Command: node ./scripts/build-registry.js
...
Error: Cannot find module
```

![Screenshot_20200720_115015](https://user-images.githubusercontent.com/10461622/87924848-bdab5100-ca7f-11ea-8e68-fc22bba9b979.png)
![Screenshot_20200720_115048](https://user-images.githubusercontent.com/10461622/87924858-c1d76e80-ca7f-11ea-9e53-4b8e8a855fb5.png)

**Comment by pedrouid** [comment]
my bad that wasn't supposed to be a postinstall... I will publish a fix in a couple of seconds

**Comment by pedrouid** [comment]
Published on v1.0.18
